### PR TITLE
Fixes #13320:  add information about releasing bastion to readme.

### PR DESCRIPTION
**Pull request by waldenraines**
http://projects.theforeman.org/issues/13320

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -293,6 +293,21 @@ To create an angular object from translated .po files run:
...
 ```
 
+## Releasing Bastion ##
+
+In order to release a new version of Bastion, the following is required:
+
+ 1. Update the version in `lib/bastion/version.rb`
+ 1. Commit the changes
+ 1. `git tag <version>`
+ 1. `git build bastion.gemspec`
```
Comment (ehelms) [inline README.md]: s/git/gem

Comment (waldenraines) [inline README.md]: Hah, oops.

**Comment by waldenraines** [comment]
@ehelms updated.

**Comment by ehelms** [comment]
ACK

**Pull request merged**